examples/nginx-tracing/Dockerfile

```
@@ -2,11 +2,11 @@
 FROM nginx:1.17.3
 
 RUN apt-get update && \
-  apt-get install -y wget tar
+  apt-get install -y wget tar jq
 
 # Install nginx-opentracing
 RUN get_latest_release() { \
-  wget -qO- "https://api.github.com/repos/$1/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'; \
+  wget -qO- "https://api.github.com/repos/$1/releases/latest" | jq -r .'tag_name'; \
```

```suggestion
  wget -qO- "https://api.github.com/repos/$1/releases/latest" | jq --raw-output .tag_name; \
```
Just so it's like https://github.com/DataDog/nginx-datadog/blob/7d2ac38f7ef3e4b97cfcf5803d1537c271343be5/example/services/nginx/install_datadog.sh#L23-L25